app/main.py

```diff
@@ -8,12 +8,13 @@ def __init__(self, name: str, age: int) -> None:
 
 
 def create_person_list(people: list) -> list:
-    _ = [Person(human["name"], human["age"]) for human in people]
+    for human in people:
+        Person(human["name"], human["age"])
 
     for human in people:
         if human.get("wife"):
             Person.people[human["name"]].wife = Person.people[human["wife"]]
         elif human.get("husband"):
-            Person.people[human["name"]
-                          ].husband = Person.people[human["husband"]]
+            Person.people[human["name"]].husband = \
+                Person.people[human["husband"]]
```

Comment:
it is better to avoid using linebreakers 
u can break this line through the brackets, but with better formating )

Reply:
In PyCharm this problem fixed automatically or no? Like if I use PyCharm instead of VSCode i won't be get this bad formatting?

Reply:
Because I have already installed Black but for him this rows not too long, and I think I must change  maybe max_length_of_row in Black to maybe flake8 max_length_of_row. But i think its not good idea because max_length_of_row depends to project style where u work, not sure)